<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayerInfoWidgetClass</class>
 <widget class="QWidget" name="GamePlayerInfoWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LobbyPlayerInfoWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_playerInfo">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>5</y>
     <width>235</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border: 0px solid gray;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_avatarBackgroundDrawing">
    <property name="geometry">
     <rect>
      <x>-17</x>
      <y>-9</y>
      <width>77</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/letter-avatar-bg-drawing.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_avatarLetter">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: rgb(255, 255, 255);
border:0px;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_surnameGivenName">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>36</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Medium</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;
color: gray;</string>
    </property>
    <property name="text">
     <string>GivenName Surname</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_avatarBackground">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>7</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Frontend.qrc">:/Frontend/assets/images/letter-avatar-bg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_username">
       <property name="font">
        <font>
         <family>Nunito Medium</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px;</string>
       </property>
       <property name="text">
        <string>@username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_pointsNumber">
       <property name="font">
        <font>
         <family>Nunito Medium</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(3, 83, 151);
color: rgb(255, 255, 255);
border-radius:7px;
boder:0px;</string>
       </property>
       <property name="text">
        <string>100 pts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_avatarBackgroundDrawing</zorder>
   <zorder>label_surnameGivenName</zorder>
   <zorder>label_avatarBackground</zorder>
   <zorder>label_avatarLetter</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Frontend.qrc"/>
 </resources>
 <connections/>
</ui>
